### PR TITLE
chore(design): add splash icons

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

**Comment by johngruen** [comment]
Fixes #70

**Comment by johngruen** [comment]
Fixes #78

File: src/styles/modules/_splash.scss [hunk 1]
```diff
@@ -82,10 +82,24 @@
...
   .splash-image {
     background: $dark;
+    background-position: center center;
     border-radius: 50%;
     height: 150px;
     margin: 14px auto 24px;
     width: 150px;
+
+    &.try {
+      @include hidpi-background-image("try", 150px 150px);
```
Comment (nchapman) [inline src/styles/modules/_splash.scss]: These are using double quotes while `respond-to` is using singles. Standardize on single?

Comment (johngruen) [inline src/styles/modules/_splash.scss]: cool, will fix and merge

**Comment by johngruen** [comment]
Fixes #70

**Pull request merged**